app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
```

Comment:
There should be a file in the views for the comments by the name of `new.html.erb` which renders a form that accepts input for creating comments. However, in your case, the file name is something else. Kindly fix it. ⬇️
![image](https://user-images.githubusercontent.com/33268927/216781308-95954ac3-29ae-4a2f-a366-5979dce6017e.png)

Reply:
@muneebulrehman I used a partial in my case to fulfill this task here is the reference to the [file](https://github.com/DaveZag/blog-app/pull/7/files#diff-2249c27c9f1ebbe020b782796c8f5f0d22f4c354e3a8a5cd50ecf8f92c96ea7fR1-R3) which is rendered without the user being redirected to a `comments/new` route on this line [here](https://github.com/DaveZag/blog-app/pull/7/files#diff-dbe3a3b30b290647eb1f4c401b675c6c0e2f130fac1cadd25c1c447417a6b94bR20).